server/app/views/admin/questions/QuestionPreview.java

```diff
@@ -46,7 +46,7 @@ public static DivTag renderQuestionPreview(
     try {
       renderedQuestion = buildQuestionRenderer(type, messages, fileUploadViewStrategy);
     } catch (UnsupportedQuestionTypeException e) {
-      renderedQuestion = div().withText(e.toString());
+      throw new RuntimeException(e);
```

Comment:
**suggestion (blocking):** This looks unrelated to implementing phone number question type. Probably better to do in a separate PR. What's motivating this change?

Reply:
This was the issue we discovered during the pair-programming session where I couldn't clearly see why the PhoneQuestion wasn't visible. You suggested to make this change to make it more obvious rather than silently failing. Since it was discovered as part of this PR, I left it here.